### PR TITLE
Fix eslint auth

### PR DESCRIPTION
**Pull request by vasavielectem**
pull request from fix -eslint-auth

### CODE REVIEW

File: Back-End/app/controllers/rack.controller.ts [hunk 1]
```diff
@@ -50,46 +54,64 @@ function createTrayObject(id, no_of_rows, no_of_columns): void {
...
-    type: sequelize.QueryTypes.SELECT
-  })
   try {
-    res.status(201).json(response)
+    if (client_fk != null) {
+      const response = await sequelize.query(query, {
+        type: sequelize.QueryTypes.SELECT
+      })
+      if (response === null) {
+        res.send("racks are null")
```
Comment (electem) [inline Back-End/app/controllers/rack.controller.ts:81]: @vasavielectem , fix the sentence

File: Back-End/app/controllers/user.controller.ts [hunk 2]
```diff
@@ -172,16 +210,16 @@ const saveClientStaff = async (req: Request, res: Response): void => {
     }
     transport.sendMail(message, function (err, info) {
       if (err) {
-        console.log(err)
+        logger.error("error in sending mail transport")
       } else {
-        console.log("mail has sent.")
-        console.log(info)
+        logger.info("mail has sent.")
+        logger.info(info)
```
Comment (electem) [inline Back-End/app/controllers/user.controller.ts:216]: Duplicate, fix it

File: Back-End/app/controllers/user.controller.ts [hunk 3]
```diff
@@ -361,8 +399,8 @@ const forgotpassword = async (req: Request, res: Response): void => {
   const data = await sequelize.query(query)
   try {
     if (data[1].rowCount >= 1) {
```
Comment (electem) [inline Back-End/app/controllers/user.controller.ts:401]: Check for null

File: Back-End/app/routes/index.ts [hunk 4]
```diff
@@ -1,19 +1,16 @@
...
-  upload,
-  getListFiles,
-  getListFilesInProfile,
-  download,
-  downloadProfileImages
-} from '../controllers/file.controller'
+import express from "express"
+//import {Router} from 'express'
+// import auth from './auth'
+//import user from './user'
```
Comment (electem) [inline Back-End/app/routes/index.ts:4]: remove commented code

File: Back-End/app/controllers/rack.controller.ts (hunk 1 above)
Comment (vasavielectem) [reply Back-End/app/controllers/rack.controller.ts]: fixed this error with correct sentence

File: Back-End/app/controllers/user.controller.ts (hunk 2 above)
Comment (vasavielectem) [reply Back-End/app/controllers/user.controller.ts]: removed duplicate code

File: Back-End/app/controllers/user.controller.ts (hunk 3 above)
Comment (vasavielectem) [reply Back-End/app/controllers/user.controller.ts]: fixed this issuess in all files

File: Back-End/app/routes/index.ts (hunk 4 above)
Comment (vasavielectem) [reply Back-End/app/routes/index.ts:4]: fixed->removed all commented code

**Pull request merged**